tests/acceptance-tests/src/acceptance/http_forwarded_for_test.go

```diff
@@ -0,0 +1,62 @@
+package acceptance_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/helpers"
+
+	"encoding/json"
+	"net"
+	"strings"
+)
+
+type ResponseJson struct {
+	Header struct {
+		X_Forwarded_For   []string `json:"X-Forwarded-For"`
+		X_Forwarded_Proto []string `json:"X-Forwarded-Proto"`
+	} `json:"Header"`
+}
+
+var headers ResponseJson
+
+var _ = BeforeEach(func() {
```

Comment:
This should be inside the `Describe` block below, otherwise it'll run before every test in the whole suite.